Stripe/PublicHeaders/STPImageLibrary.h

```diff
@@ -47,6 +47,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (UIImage *)masterCardCardImage;
 
+/**
+ An icon representing UnionPay.
+ */
++ (UIImage *)unionPayCardImage;
```

Comment:
should we support a localized UnionPay icon? looks like stripe.js has a chinese variant
https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/a2331833c7a81742e29c08c387ab18b5f804d188/src/three_ds/inner/brands.js#L12

Reply:
Looks like we should support the chinese icon if `[[NSLocale currentLocale] languageCode] == @"zh"` to match the web logic